avocado/utils/vmimage.py

```diff
@@ -348,8 +352,11 @@ def get(name=None, version=None, build=None, arch=None, checksum=None,
                      download.
     :param algorithm: (optional) Hash type, used when the checksum is
                       provided.
-    :param cache_dir: (optional) Local system path where the images and
-                      the snapshots will be held.
+    :param cache_dir: (optional) Local system path where the base
+                      images will be held.
+    :param snapshot_dir: (optional) Local system path where the snapshot images
```

Comment:
Is the description of snapshot_dir parameter correct? `where the snapshot images the will be held` seems strange.

Reply:
It seems correct to me.  What happens here is that the base image will be downloaded, by default, on a "cache dir".  Then, when the vmimage is handled to the user, it's actually a snapshot image, that will contain only the differences from the main one.  "will be held" or "will be kept" is how I'd put it.  Do you have a suggestion?

Reply:
Sorry I was not clear. The description is:
`where the snapshot images the will be held`
I think it would be:
`where the snapshot images will be held`